common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -86,6 +86,15 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+  val LiveblogDynamicAdvertsSwitch = Switch(
+    "Commercial",
+    "liveblog-dynamic-adverts",
+    "Dynamically insert inline adverts on liveblogs",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 2, 14),
```

Comment:
No way it's Sunday! There is a [norm](https://en.wikipedia.org/wiki/ISO_8601) for that! :wink: